conference.js

```diff
@@ -629,47 +630,63 @@ export default {
+                            audioAndVideoError = err;
 
-                        return (
-                            createLocalTracksF({ devices: [ 'audio' ] }, true));
-                    } else if (requestedAudio && !requestedVideo) {
+                            return (
+                                createLocalTracksF({ devices: [ 'audio' ] }, true));
+                        } else if (requestedAudio && !requestedVideo) {
+                            audioOnlyError = err;
+
+                            return [];
+                        } else if (requestedVideo && !requestedAudio) {
+                            videoOnlyError = err;
+
+                            return [];
+                        }
+                        logger.error('Should never happen');
+                    })
+                    .catch(err => {
+                        // Log this just in case...
+                        if (!requestedAudio) {
+                            logger.error('The impossible just happened', err);
+                        }
                         audioOnlyError = err;
 
-                        return [];
-                    } else if (requestedVideo && !requestedAudio) {
+                        // Try video only...
+                        return requestedVideo
+                            ? createLocalTracksF({ devices: [ 'video' ] }, true)
+                            : [];
+                    })
+                    .catch(err => {
+                        // Log this just in case...
+                        if (!requestedVideo) {
+                            logger.error('The impossible just happened', err);
+                        }
                         videoOnlyError = err;
 
                         return [];
-                    }
-                    logger.error('Should never happen');
-                })
-                .catch(err => {
-                    // Log this just in case...
-                    if (!requestedAudio) {
-                        logger.error('The impossible just happened', err);
-                    }
-                    audioOnlyError = err;
+                    });
 
-                    // Try video only...
-                    return requestedVideo
-                        ? createLocalTracksF({ devices: [ 'video' ] }, true)
-                        : [];
-                })
-                .catch(err => {
-                    // Log this just in case...
-                    if (!requestedVideo) {
-                        logger.error('The impossible just happened', err);
-                    }
-                    videoOnlyError = err;
+            tryCreateLocalTracks = this._videoBlurScriptPromise
+                .then(() =>
+                    _createLocalTracks([ getJitsiMeetGlobalNS().effects.JitsiStreamBlurEffect ])
+                )
+                .catch(error => {
+                    APP.store.getState()['features/blur'].blurEnabled = false;
```

Comment:
OK I don't know what is the intention here but this is not going to work. You should use an action to change the store!

Reply:
Removed the code